main.go

```diff
@@ -195,6 +201,19 @@ func main() {
 		reporter.Report()
 	}
 
+	ifSendToBC := defaultIfBlank(os.Getenv("ANODOT_SEND_TO_BC"), "true")
+	sendToBCPeriod, err := strconv.Atoi(defaultIfBlank(os.Getenv("ANODOT_SEND_TO_BC_PERIOD_SEC"), "60"))
+	if err != nil {
+		log.Fatalf("Could not parse ANODOT_SEND_TO_BC_PERIOD_SEC: %v", err)
+	}
+	if ifSendToBC != "false" {
```

Comment:
may be move 
`accessKey := os.Getenv("ANODOT_ACCESS_KEY")
	if len(strings.TrimSpace(accessKey)) == 0 {
		log.Fatalf("ANODOT_ACCESS_KEY is not specified")
	}`

here ? so we will require only token when ANODOT_SEND_TO_BC enabled